src/OSSupport/Network.h

```diff
@@ -0,0 +1,215 @@
+
+// Network.h
+
+// Declares the classes used for the Network API
+
+
+
+
+
+#pragma once
+
+
+
+
+
+/** Interface that provides the methods available on a single TCP connection. */
+class cTCPLink
```

Comment:
Its more about splitting up the cpp file than the h. If you split off the interface, you can split the implementation off in to the same cpp as the interface.